.travis.yml

```diff
@@ -4,17 +4,39 @@ go: "1.10"
 services:
 - docker
 
-install: true
+install:
+   - make manifests
+
 script: make container-build 
 
+before_deploy:
+- docker login -u="$DOCKER_USER" -p="$DOCKER_PASS" quay.io
```

Comment:
you should rather use this syntax:
 echo "$DOCKER_PASS" | docker login -u="$DOCKER_USER" quay.io --password-stdin
Usage of -p is insecure in cli.